code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -568,7 +568,7 @@
 		/obj/machinery/vending/modularpc = "Deluxe Silicate Selections",
 		/obj/machinery/vending/tool = "YouTool",
 		/obj/machinery/vending/custom = "Custom Vendor",
-		/obj/machinery/vending/dorms = "KinkVend")	//SKYRAT EDIT ADDITION
+		/obj/machinery/vending/lustwish = "LustWish")	//LAMELLA ERP UPDATE ADDITION
```

Comment:
ALL INSTANCES OF THIS ARE TO BE UNDONE AND REPLACED WITH //SKYRAT EDIT ADDITION. THIS IS NON-NEGOTIABLE.

IN THE EVENT OF A CONFLICT, THESE WILL NOT BE CARRIED OVER.